### ISSUE TITLE
Question about the locality

### ISSUE THREAD

**Issue by sev777** [VLKEB/VLKEB#12]
The locality is calculated by: 
       
             if 'locality_output' in metrics['post'].keys():
                      assert len(metrics['post']['locality_output']) == \
                              len(metrics['pre']['locality_output'])
                      metrics['post']['locality_acc'] = \
                          np.mean(np.equal(metrics['post']['locality_output'],
                                              metrics['pre']['locality_output']))
                      metrics['post'].pop('locality_output')
                      metrics['pre'].pop('locality_output')
                      

                if 'multimodal_locality_output' in metrics['post'].keys():
                    assert len(metrics['post']['multimodal_locality_output']) == \
                            len(metrics['pre']['multimodal_locality_output'])
                    metrics['post']['multimodal_locality_acc'] = \
                        np.mean(np.equal(metrics['post']['multimodal_locality_output'],
                                            metrics['pre']['multimodal_locality_output']))
                    metrics['post'].pop('multimodal_locality_output')
                    metrics['pre'].pop('multimodal_locality_output')

But the metrics['post']['multimodal_locality_output'] or metrics['post']['locality_output'] contain 2 elements (pred_ids and targ ), due to the return value of def compute_multimodal_edit_quality() -> return acc, (pred_ids.numpy(), targ.numpy()).

So the locality value at least is 0.5 due to there are targ in the list. Is it right?

**Comment by hanhuang22** [comment]
Hi, the locality for ike is here: https://github.com/VLKEB/VLKEB/blob/10951b7b3788928f578b73f07eac9e1eaa0316f3/easyeditor/evaluate/evaluate.py#L329-L339

and for other methods is here:
https://github.com/VLKEB/VLKEB/blob/10951b7b3788928f578b73f07eac9e1eaa0316f3/easyeditor/trainer/MultimodalTrainer.py#L167-L169

I think the codes you referenced are not used and the locality value is not at least 0.5.

**Comment by sev777** [comment]
Thanks,  but the code in 'https://github.com/VLKEB/VLKEB/blob/10951b7b3788928f578b73f07eac9e1eaa0316f3/multimodal_edit.py#L126'  seems use the different codes.

` metrics, edited_model, _ = editor.edit_dataset(
        ds=eval_ds,
        train_ds='train_ds',
        keep_original_weight=True        
    )`

**Comment by hanhuang22** [comment]
For IKE, in edit_dataset function:
https://github.com/VLKEB/VLKEB/blob/10951b7b3788928f578b73f07eac9e1eaa0316f3/easyeditor/editors/multimodal_editor.py#L448

and then 
https://github.com/VLKEB/VLKEB/blob/10951b7b3788928f578b73f07eac9e1eaa0316f3/easyeditor/editors/multimodal_editor.py#L473

after this, go to
https://github.com/VLKEB/VLKEB/blob/10951b7b3788928f578b73f07eac9e1eaa0316f3/easyeditor/editors/multimodal_editor.py#L492

finally the aforementioned function:
https://github.com/VLKEB/VLKEB/blob/10951b7b3788928f578b73f07eac9e1eaa0316f3/easyeditor/evaluate/evaluate.py#L260

In this function, the metrics are calculated.

**Comment by sev777** [comment]
Hi， I want to known about the LLava1.5 7B,  the 40G A100 couldn't run the MEND? And I find when evaluate with MEND-llava, the GPU memory will keep increase for each edits, did you meet same issue?  Thanks!
And in llava_llama.py, during training MEND, the following information will occur, did you just ignore it or need to made some change?
`         if torch.isnan(outputs.logits).any():
            print("LLaVA logits has nan!!!!!!!!!!!!!!!")`
